tests/appsec/waf/test_addresses.py

```diff
@@ -322,8 +322,10 @@ class Test_PathParams(BaseTestCase):
     """Appsec supports values on server.request.path_params"""
 
     @bug(library="dotnet", reason="attack is not reported")
+    @missing_feature(context.library < "java@0.99.0" and context.weblog_variant in ["vertx3", "ratpack"])
```

Comment:
I guess it's a rebase issue -> this line can be removed

Reply:
```suggestion
```